public/index.html

```diff
@@ -24,12 +22,22 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>MyNotif</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <script src="https://cdn.onesignal.com/sdks/OneSignalSDK.js" defer></script>
```

Comment:
Wait don't we have a better way to integrate that with React?

Reply:
I've integrate via yarn add